assets/src/edit-story/app/highlights/quickActions/useQuickActions.js

```diff
@@ -248,12 +248,11 @@ const useQuickActions = () => {
         onClick: handleFocusLinkPanel(selectedElement?.id),
         ...actionMenuProps,
       },
-      {
+      selectedElementAnimations?.length && {
         Icon: Eraser,
         label: ACTION_TEXT.CLEAR_ANIMATIONS,
         onClick: () => handleClearAnimations(selectedElement?.id),
         separator: 'top',
-        disabled: !selectedElementAnimations?.length,
         ...actionMenuProps,
       },
```

Comment:
💭 I think we should only add this element in if it's supposed to be there. With this our return value would look like
```js
[{}, {}, {}, 0]
```

So maybe this could be done like this instead:

```js
  const foregroundCommonActions = useMemo(
    () => {
    const baseActions = [
      {
        Icon: CircleSpeed,
        label: ACTION_TEXT.ADD_ANIMATION,
        onClick: handleFocusAnimationPanel(selectedElement?.id),
        ...actionMenuProps,
      },
      {
        Icon: Link,
        label: ACTION_TEXT.ADD_LINK,
        onClick: handleFocusLinkPanel(selectedElement?.id),
        ...actionMenuProps,
      },
    ];

    const clearAction =  {
        Icon: Eraser,
        label: ACTION_TEXT.CLEAR_ANIMATIONS,
        onClick: () => handleClearAnimations(selectedElement?.id),
        separator: 'top',
        ...actionMenuProps,
      };

    return selectedElementAnimations?.length ? [...baseActions, clearAction] : baseActions;
    },
    [
      handleClearAnimations,
      handleFocusLinkPanel,
      handleFocusAnimationPanel,
      actionMenuProps,
      selectedElement?.id,
      selectedElementAnimations?.length,
    ]
  );
```

Then the only elements of the array are objects that should be rendered.

Reply:
🤦  whoops, yup